<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InspectorLevelsWidget</class>
 <widget class="QWidget" name="InspectorLevelsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>231</width>
    <height>312</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutMinIn">
     <item>
      <spacer name="horizontalSpacerMinIn">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>31</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelMinIn">
       <property name="text">
        <string>Min in</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="sliderMinIn">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxMinIn">
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutMaxIn">
     <item>
      <spacer name="horizontalSpacerMaxIn">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>29</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelMaxIn">
       <property name="text">
        <string>Max in</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="sliderMaxIn">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxMaxIn">
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutMinOut">
     <item>
      <spacer name="horizontalSpacerMinOut">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>23</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelMinOut">
       <property name="text">
        <string>Min out</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="sliderMinOut">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxMinOut">
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutMaxOut">
     <item>
      <spacer name="horizontalSpacerMaxOut">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>21</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelMaxOut">
       <property name="text">
        <string>Max out</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="sliderMaxOut">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxMaxOut">
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutGamma">
     <item>
      <spacer name="horizontalSpacerGamma">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>22</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelGamma">
       <property name="text">
        <string>Gamma</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="sliderGamma">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxGamma">
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutTransitionDuration">
     <item>
      <spacer name="horizontalSpacerTransitionDuration">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>17</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelTransitionDuration">
       <property name="text">
        <string>Duration</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxTransitionDuration">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelFrm1">
       <property name="text">
        <string>frm</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutTween">
     <item>
      <spacer name="horizontalSpacerTween">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>30</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelEasing">
       <property name="text">
        <string>Tween</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxTween">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutDefer">
     <item>
      <spacer name="horizontalSpacerDefer1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>29</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelDefer">
       <property name="text">
        <string>Defer</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxDefer">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacerDefer2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacerLevels">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>spinBoxTransitionDuration</sender>
   <signal>valueChanged(int)</signal>
   <receiver>InspectorLevelsWidget</receiver>
   <slot>transitionDurationChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>171</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxTween</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>InspectorLevelsWidget</receiver>
   <slot>tweenChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderMinIn</sender>
   <signal>valueChanged(int)</signal>
   <receiver>InspectorLevelsWidget</receiver>
   <slot>sliderMinInChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderMaxIn</sender>
   <signal>valueChanged(int)</signal>
   <receiver>InspectorLevelsWidget</receiver>
   <slot>sliderMaxInChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>9</x>
     <y>4</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderMinOut</sender>
   <signal>valueChanged(int)</signal>
   <receiver>InspectorLevelsWidget</receiver>
   <slot>sliderMinOutChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>18</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderMaxOut</sender>
   <signal>valueChanged(int)</signal>
   <receiver>InspectorLevelsWidget</receiver>
   <slot>sliderMaxOutChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>14</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxMinIn</sender>
   <signal>valueChanged(int)</signal>
   <receiver>InspectorLevelsWidget</receiver>
   <slot>spinBoxMinInChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxMaxIn</sender>
   <signal>valueChanged(int)</signal>
   <receiver>InspectorLevelsWidget</receiver>
   <slot>spinBoxMaxInChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxMinOut</sender>
   <signal>valueChanged(int)</signal>
   <receiver>InspectorLevelsWidget</receiver>
   <slot>spinBoxMinOutChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxMaxOut</sender>
   <signal>valueChanged(int)</signal>
   <receiver>InspectorLevelsWidget</receiver>
   <slot>spinBoxMaxOutChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxGamma</sender>
   <signal>valueChanged(int)</signal>
   <receiver>InspectorLevelsWidget</receiver>
   <slot>spinBoxGammaChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderGamma</sender>
   <signal>valueChanged(int)</signal>
   <receiver>InspectorLevelsWidget</receiver>
   <slot>sliderGammaChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>14</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxDefer</sender>
   <signal>stateChanged(int)</signal>
   <receiver>InspectorLevelsWidget</receiver>
   <slot>deferChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>resetMinIn(QString)</slot>
  <slot>sliderMinInChanged(int)</slot>
  <slot>sliderMinOutChanged(int)</slot>
  <slot>resetMinOut(QString)</slot>
  <slot>sliderMaxInChanged(int)</slot>
  <slot>resetMaxIn(QString)</slot>
  <slot>sliderMaxOutChanged(int)</slot>
  <slot>resetMaxOut(QString)</slot>
  <slot>sliderGammaChanged(int)</slot>
  <slot>resetGamma(QString)</slot>
  <slot>resetDuration(QString)</slot>
  <slot>resetTween(QString)</slot>
  <slot>transitionDurationChanged(int)</slot>
  <slot>tweenChanged(QString)</slot>
  <slot>spinBoxMinInChanged(int)</slot>
  <slot>spinBoxMinOutChanged(int)</slot>
  <slot>spinBoxMaxInChanged(int)</slot>
  <slot>spinBoxMaxOutChanged(int)</slot>
  <slot>spinBoxGammaChanged(int)</slot>
  <slot>resetDefer(QString)</slot>
  <slot>deferChanged(int)</slot>
 </slots>
</ui>
